### ISSUE TITLE
Create FFForest

### ISSUE THREAD

**Issue by ndphillips** [ndphillips/FFTrees#15]
Create an `FFForest`, a function that randomly samples data from a training set `data` and generates FFTrees from each sample. Then, decisions from all trees should be combined to create probabilistic predictions for a test set.

A version of this is currently being implemented as simFFTrees

**Comment by ndphillips** [comment]
Partially done.

Now add `predict.FFForest()`, which takes fits from all the trees in the forest, and returns the proportion of trees that classified each exemplar, as well as a majority vote (for class decisions)
